Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -1594,16 +1594,15 @@ void RenderLayer::setAncestorChainHasVisibleDescendant()
 void RenderLayer::updateAncestorDependentState()
 {
 #if ENABLE(LAYER_BASED_SVG_ENGINE)
-    m_enclosingSVGResourceContainer = nullptr;
-    m_enclosingSVGHiddenContainer = nullptr;
+    m_enclosingSVGHiddenOrResourceContainer = nullptr;
     auto determineSVGAncestors = [&] (const RenderElement& renderer) {
         for (auto* ancestor = renderer.parent(); ancestor; ancestor = ancestor->parent()) {
             if (auto* container = dynamicDowncast<RenderSVGResourceContainer>(ancestor)) {
-                m_enclosingSVGResourceContainer = container;
+                m_enclosingSVGHiddenOrResourceContainer = container;
                 return;
             }
```

Comment:
Agreed, fixing this in https://github.com/WebKit/WebKit/pull/19826.

Reply:
This is not the case, dynamicDowncast fails here!

Reply:
What a dilemma:

From RenderObject.h:

```
    bool isSVGHiddenContainer() const { return type() == Type::SVGHiddenContainer; }
    virtual bool isSVGResourceContainer() const { return false; }
....
```

is<RenderSVGHiddenContainer> will return false for renderer that derives from RenderSVGHiddenContainer.
That is because the type() is not 'HiddenContainer', but something more specific for these renderers, e.g. 'SVGResourceClipper'.

@shallawa - can you confirm?

Reply:
I talked to @cdumez and he said he is going to fix it to work as before.